packages/commons/src/events/ActionDocument.ts

```diff
@@ -40,6 +41,7 @@ export type ActionStatus = keyof typeof ActionStatus
 export const ActionBase = z.object({
   id: z.string(),
   transactionId: z.string(),
+  createdByUserType: TokenUserType.optional(),
```

Comment:
Should we have these as mandatory? Is there a use case for it being optional?

Reply:
I made it required again and it worked this time.  I must have made a different mistake before.